lib/dynflow/rails/configuration.rb

```diff
@@ -100,8 +100,13 @@ def increase_db_pool_size?
       end
 
       def calculate_db_pool_size(world)
-        self.db_pool_size || world.config.queues.values.inject(5) do |pool_size, pool_options|
-          pool_size += pool_options[:pool_size]
+        base_value = 5
+        if defined?(::Sidekiq)
+          Sidekiq.options[:concurrency] + base_value
+        else
+          self.db_pool_size || world.config.queues.values.inject(base_value) do |pool_size, pool_options|
+            pool_size += pool_options[:pool_size]
+          end
```

Comment:
if i'm reading this right, it means with sidekiq, we can no longer configure the db_size (independently of sidekiq's concurrency setting).  Do you see anyone ever needing to do that?

Reply:
That wasn't intentional, I'll put it back. It served us well in the past